x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/event_column_view.test.tsx

```diff
@@ -134,41 +134,6 @@ describe('EventColumnView', () => {
     expect(wrapper.find('[data-test-subj="pin"]').exists()).toBe(false);
   });
 
-  test('it render AddToCaseAction if timelineId === TimelineId.detectionsPage', () => {
```

Comment:
moved to `.../detections/components/alerts_table/timeline_actions/alert_context_menu.test.tsx`